### PR TITLE
fix(FEV-1386): Navigation panel is rendered 'alongSide' when set to 'over' when dual-screen plugin is active

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves: FEV-1386

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/pip/pip-parent.tsx
Comment (semarche-kaltura) [inline src/components/pip/pip-parent.tsx:49]: remove also `playerHeight` and `playerWidth` from mapStateToProps and component props.

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/components/pip/pip-parent.tsx
Comment (lianbenjamin) [inline src/components/pip/pip-parent.tsx:49]: @JonathanTGold removing height for playerContainer will cause the dual to stretch on the main player. see screenshot-
image
you have to keep the height.

**Review by lianbenjamin — CHANGES_REQUESTED** [review]

File: src/components/pip/pip-parent.tsx
Comment (JonathanTGold) [reply src/components/pip/pip-parent.tsx:45]: done

Comment (semarche-kaltura) [reply src/components/pip/pip-parent.tsx:45]: @JonathanTGold the PR approved, please be sure you merged changes from latest master into current branch before merge the PR. Lian made some changes related to video-container size, so we don't want get any regression for the component.

Comment (semarche-kaltura) [reply src/components/pip/pip-parent.tsx:45]: @lianbenjamin please, review the PR.

**Review by semarche-kaltura — APPROVED** [review]

**Review by lianbenjamin — APPROVED** [review]

File: src/components/pip/pip-parent.tsx
Comment (lianbenjamin) [reply src/components/pip/pip-parent.tsx:45]: approved - when i merged the changes to dualscreen related to ratio of the pip player, i tested it with current PR changes, to make sure they do not collide. @JonathanTGold please double check and make sure both functionalities work as expected against latest code.

**Pull request merged**
